src/Uno.UI/UI/Xaml/UIElement.net.cs

```diff
@@ -48,5 +48,22 @@ internal Windows.Foundation.Point GetPosition(Point position, global::Windows.UI
 		protected virtual void OnVisibilityChanged(Visibility oldValue, Visibility newVisibility)
 		{
 		}
+
+		public GeneralTransform TransformToVisual(UIElement visual)
+		{
+			// stub
+
+			return new MatrixTransform
+			{
+				Matrix = new Matrix(
+					m11: 1,
+					m12: 0,
+					m21: 0,
+					m22: 1,
+					offsetX: 0, //TODO
+					offsetY: 0  //TODO
+				)
```

Comment:
`TODO` are not just for `Offset<X|Y>` ;)

Reply:
Fair enough, but they're TODO in that case for all platforms, not just unit tests :p

Reply:
No, the `TransformToVisual` is implemented now on iOS, Android and WASM 🤔

Reply:
Ok, I guess I can't see where they're setting the non-translate matrix components then.

https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.Android.cs#L71
https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.iOS.cs#L142
https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.wasm.cs#L618

Reply:
Hummm right ... I thought the `TransformBounds`, but it's on the `GeneralTransform` (https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Media/GeneralTransform.cs#L31) ... so it means that my next bug will be harder to fix 😥